### ISSUE TITLE
Error: Unexpected Paths in Downloaded Kata Containers Release Archive

### ISSUE THREAD

**Issue by jena-satyabrata** [kata-containers/kata-containers#10575]
# Get your issue reviewed faster

To help us understand the problem more quickly, please do the following:

1. Ensure that you've reproduced the issue with the [latest release](https://github.com/kata-containers/kata-containers/releases).
1. Run the `kata-collect-data.sh` script, which is installed as part of Kata Containers
   or `kata-containers.collect-data`, which is installed as part of the Kata Containers
   snapcraft package.
   ```sh
   $ sudo kata-collect-data.sh > /tmp/kata.log
   ```
   or
   ```sh
   $ sudo kata-containers.collect-data > /tmp/kata.log
   ```
1. Review the output file (`/tmp/kata.log`) to ensure it doesn't
   contain any private / sensitive information
1. Paste the *entire* contents of the file into this issue as a comment
   (the script generates markdown format output).

The information provided will help us to understand the problem more quickly
so saves time for both of us! :smile:

# Description of problem

Description: I encountered an issue while trying to install Kata Containers using the kata-manager.sh script. The script fails with an error indicating that the downloaded release archive contains unexpected paths.
Steps to Reproduce:
Run the following command to execute the installation script
bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -N -f"

# Expected result

(replace this text with an explanation of what you thought would happen)

# Actual result

bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -N -f"
INFO: Containerd will be installed during the nerdctl installation ('-c' option ignored)
INFO: Checking dependencies
INFO: Downloading Kata Containers release (latest version)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  399M  100  399M    0     0  10.6M      0  0:00:37  0:00:37 --:--:-- 10.9M
INFO: Checking file '/tmp/tmp.co1qIv2mUC/kata-static-3.11.0-amd64.tar.xz'
ERROR: File '/tmp/tmp.co1qIv2mUC/kata-static-3.11.0-amd64.tar.xz' contains unexpected paths: './pause_bundle/
./pause_bundle/config.json
./pause_bundle/sha256_b42c514302e917881d20666a5990795df507ec14b2d79fdb1e41a619e66b77b6.mtree
./pause_bundle/rootfs/
./pause_bundle/rootfs/pause
./usr/

# Further information

I have verified that the download was not corrupted.
I have tried using the latest version of the kata-manager.sh script.
The issue persists across multiple attempts.

# Kata Containers survey

Please consider taking the survey to help us help you: https://openinfrafoundation.formstack.com/forms/kata_containers_user_survey

**Comment by chenxingbingmail** [comment]
I encountered  the same issue.
root@iZ2ze1cyvz1p76pvqk5l6vZ:~# ./kata-manager.sh
INFO: Checking dependencies
INFO: Running pre-checks
INFO: Downloading Kata Containers release (latest version)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  399M  100  399M    0     0   274k      0  0:24:48  0:24:48 --:--:--  231k
INFO: Checking file '/tmp/tmp.XKHspOacxV/kata-static-3.11.0-amd64.tar.xz'
ERROR: File '/tmp/tmp.XKHspOacxV/kata-static-3.11.0-amd64.tar.xz' contains unexpected paths: './pause_bundle/
./pause_bundle/config.json
./pause_bundle/sha256_b42c514302e917881d20666a5990795df507ec14b2d79fdb1e41a619e66b77b6.mtree
./pause_bundle/rootfs/
./pause_bundle/rootfs/pause
./usr/
./usr/local/
./usr/local/bin/
./usr/local/bin/attestation-agent
./usr/local/bin/confidential-data-hub
./usr/local/bin/luks-encrypt-storage
./usr/local/bin/api-server-rest
./usr/bin/
./usr/bin/kata-agent
./usr/lib/
./usr/lib/systemd/
./usr/lib/systemd/system/
./usr/lib/systemd/system/kata-agent.service
./usr/lib/systemd/system/kata-containers.target'

**Comment by fidencio** [comment]
@stevenhorsman, this seems to be a side effect of having all the artefacts as part of the release, as now all of them end up as part of the released tarball.

**Comment by stevenhorsman** [comment]
Yeah - as we discussed it seems like we might need to find a way to decouple the release tarball and artifacts used in the build 🤔

**Comment by hulk-zhk** [comment]
I also encountered the same problem, is there any solution?

**Comment by stevenhorsman** [comment]
> I also encountered the same problem, is there any solution?

This should be fixed in the 3.12 release that is due to merge any day
